.circleci/config.yml

```diff
@@ -237,49 +237,49 @@ commands:
             . venv/bin/activate
             twine upload dist/*
 
-  do_docker_login:
-    description: "Login to docker"
-    parameters:
-      use_ecr:
-        type: boolean
-        default: false
-    steps:
-      - when:
-          condition: << parameters.use_ecr >>
-          steps:
-            - run:
-                name: docker login with ECR
-                command: |
-                  apk add python3 py3-pip
-                  python3 -m pip install awscli
-                  aws ecr get-login-password | docker login --username AWS --password-stdin $AWS_ECR_REGISTRY
-      - unless:
-          condition: << parameters.use_ecr >>
-          steps:
-            - run:
-                name: docker login
-                command: |
-                  docker login --username "${DOCKER_USERNAME}" --password "${DOCKER_PASSWORD}"
-
-  do_docker_bash:
-    description: "Add bash"
+  
+  do_docker_prep:
+    description: "Install packages and set commit."
     steps:
       - run:
-          name: install bash
+          name: install packages
+          command: |
+            apk add python3 py3-pip git cosign=2.0.1-r2 bash
```

Comment:
do we need to add a ticket to un-pin this once they release?

Reply:
No - that is the alpine 2.0.1 release identifier.
We could move it to `cosign~2.` to indicate any 2.x.x release.